redis_checkpoint_test.go

```diff
@@ -0,0 +1,50 @@
+package connector
+
+import (
+	"testing"
+
+	"gopkg.in/redis.v5"
+)
+
+var defaultAddr = "127.0.0.1:6379"
+
+func Test_CheckpointLifecycle(t *testing.T) {
+	client := redis.NewClient(&redis.Options{Addr: defaultAddr})
+
+	c := RedisCheckpoint{
+		AppName:    "app",
+		StreamName: "stream",
+		client:     client,
+	}
+
+	// set checkpoint
+	c.SetCheckpoint("shard_id", "testSeqNum")
+
+	// checkpoint exists
+	if val := c.CheckpointExists("shard_id"); val != true {
+		t.Fatalf("checkpoint exists expected true, got %s", val)
```

Comment:
CheckpointExists returns bool, so `val` will be bool. Can we use %t verb to format print on bool correctly? =))